forge/src/main/java/org/openjdk/skara/forge/gitlab/GitLabRepository.java

```diff
@@ -132,6 +132,15 @@ private boolean hasHeadHash(JSONValue json) {
 
     @Override
     public List<PullRequest> pullRequests() {
+        return request.get("merge_requests")
```

Comment:
We need a maxPages limit here too. I see it's missing for pullRequestsAfter for Gitlab as well, which is a bug and should also be fixed.

Reply:
Fixed.